mesonbuild/backend/ninjabackend.py

```diff
@@ -2322,6 +2323,7 @@ def generate_link(self, target, outfile, outname, obj_list, linker, extra_args=[
         return elem
 
     def determine_rpath_dirs(self, target):
+        current_dir = self.get_target_dir(target)
```

Comment:
__[flake8]__
* [F841] local variable 'current_dir' is assigned to but never used

<!-- Comment by [SideCI](https://sideci.com) -->